MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDelegate.java

```diff
@@ -95,7 +95,7 @@ public float getElevation() {
      * @see View#setElevation(float)
      */
     public void setElevation(float elevation) {
-        boolean needsWrap = elevation > mElevation;
+        boolean needsWrap = elevation != mElevation;
```

Comment:
The issue (after fixing a crash) was in a "growing" view every time we remove elevation. Its padding was set to a bigger one every time elevation goes to 0.